docs/src/examples/components/Accordion/Variations/AccordionNestedExample.shorthand.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Accordion } from '@stardust-ui/react'
+
+const panels = [
+  {
+    key: 'p',
+    title: 'What is a point?',
+    content: 'Use the haptic SDD circuit, then you can index the redundant pixel!',
+  },
+  {
+    key: 'd',
+    title: 'What is a dimension of a point?',
+    content: 'We need to copy the primary USB firewall!',
+  },
+  {
+    key: 'e',
+    title: 'Nested Accordion',
+    accordion: {
```

Comment:
do we really need to support this prop? Would the following shorthand structure work for us:

```js
const nestedPanels = [
  { key: ..., ... }
  ...
]

panels = [
  { key: ..., ... },
  { key: ..., content: <Accordion panels={nestedPanels} ... /> }
]
```